src/main/java/org/hydev/mcpm/client/arguments/parsers/ExportPluginsParser.java

```diff
@@ -0,0 +1,39 @@
+package org.hydev.mcpm.client.arguments.parsers;
+
+import net.sourceforge.argparse4j.inf.Namespace;
+import net.sourceforge.argparse4j.inf.Subparser;
+import net.sourceforge.argparse4j.inf.Subparsers;
+import org.hydev.mcpm.client.commands.entries.ExportPluginsController;
+import org.hydev.mcpm.client.database.inputs.ExportPluginsInput;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.OutputStream;
+import java.util.function.Consumer;
+
+/**
+ * Parser for the ExportPluginsBoundary interface.
+ */
+public class ExportPluginsParser implements CommandParser {
+    private ExportPluginsController controller;
+
+    public ExportPluginsParser(ExportPluginsController controller) {
+        this.controller = controller;
+    }
+
+    @Override
+    public @Nullable Subparser configure(Subparsers parsers) {
+        var parser = parsers.addParser("export"); // name of command
+
+        parser.addArgument("outfile") // add optional output file
+                .type(OutputStream.class).dest("outfile"); // of type OutputStream
+        parser.addArgument("-c", "--cache")
+                .type(boolean.class).dest("cache");
```

Comment:
> `boolean.class`

This is pretty cool that it builds. I would expect `Boolean.class`.

Reply:
surely it works...